lib/knapsack/presenter.rb

```diff
@@ -51,10 +51,15 @@ def time_offset_warning
         }
         if Knapsack.tracker.time_exceeded?
           str << %{
-Tests on this CI node took more than max allowed node time execution.
+Tests on this CI node took more than the max allowed node time execution.
```

Comment:
idea for lingo improvements:
`Test on this CI node ran for longer than the max allowed node time execution.`